src/approval/ApprovalController.test.ts

```diff
@@ -1,24 +1,50 @@
 import { errorCodes } from 'eth-rpc-errors';
 import sinon from 'sinon';
-import { ApprovalController } from './ApprovalController';
+import { ControllerMessenger } from '../ControllerMessenger';
+import {
+  ApprovalController,
+  ApprovalControllerActions,
+  ApprovalControllerEvents,
+  ApprovalControllerMessenger,
+} from './ApprovalController';
 
 const STORE_KEY = 'pendingApprovals';
 
 const TYPE = 'TYPE';
 
-const defaultConfig = {
-  showApprovalRequest: () => undefined,
-};
+const controllerName = 'ApprovalController';
+
+function getRestrictedMessenger() {
+  // The 'Other' types are included to demonstrate that this all works with a
```

Comment:
There don't appear to be any other types here :sweat_smile: So this comment is not accurate. You could delete it or add the dummy types.

It seemed useful to have secondary types when the controller messenger was still a work in progress, to demonstrate how it worked. It's less important now that we have working examples in prod. But I guess it doesn't hurt either.

Reply:
Hm, I'm going to delete the comment. I think the controller messenger or base controller tests would be a better place demonstrate how the types work.

Reply:
#565